pkg/virtiofs/virtiofs.go

```diff
@@ -51,3 +52,19 @@ func GetFilesystemPersistentVolumes(vmi *v1.VirtualMachineInstance) []v1.Volume
 func HasFilesystemPersistentVolumes(vmi *v1.VirtualMachineInstance) bool {
 	return len(GetFilesystemPersistentVolumes(vmi)) > 0
 }
+
+func FSMountPoint(volume *v1.Volume) string {
+	volumeMountPoint := fmt.Sprintf("/%s", volume.Name)
+
+	if volume.ConfigMap != nil {
+		volumeMountPoint = config.GetConfigMapSourcePath(volume.Name)
+	} else if volume.Secret != nil {
+		volumeMountPoint = config.GetSecretSourcePath(volume.Name)
+	} else if volume.ServiceAccount != nil {
+		volumeMountPoint = config.ServiceAccountSourceDir
+	} else if volume.DownwardAPI != nil {
+		volumeMountPoint = config.GetDownwardAPISourcePath(volume.Name)
+	}
```

Comment:
nit: can we please here convert this to a switch case

Reply:
I'm in the minority that don't like switch cases :-) , but I'll do you are right

Reply:
I'm very allergic to `else if`, so, I cannot join you in this :D